Assignment2/Question1.java

```diff
@@ -0,0 +1,138 @@
+package Assignment2;
+
+import java.util.Stack;
+
+public class Question1<T> {
+	public static Stack<Node> path = new Stack<Node>();

```

Comment:
Having global static variables for path is a bit ugly. Can you think of a nicer more object oriented way of doing this?

One way would be to have determinePath() return the path, but there are others.